packages/tailwindcss/src/utilities.ts

```diff
@@ -1384,6 +1420,20 @@ export function createUtilities(theme: Theme) {
     },
   ])
 
+  /**
+   * @css `perspective`
+   */
+  staticUtility('perspective-none', [['perspective', 'none']])
+  functionalUtility('perspective', {
+    themeKeys: ['--perspective'],
+    handle: (value) => [decl('perspective', value)],
+  })
+  suggest('perspective', () => [
+    {
+      valueThemeKeys: ['--perspective'],
+    },
+  ])
```

Comment:
You should be able to drop this `suggest` line since we don't support bare values or modifiers.

Reply:
Ah, gotcha. I thought this was required for the theme keys, but I see `functionalUtility` handles that.